File: sdk/javascript/packages/oracle/src/OciKmsClient.ts
import { KmsCryptoClient } from "oci-keymanagement/lib/client";
import { OCISessionConfig } from "./OciSessionConfig";

export class OciKmsClient {
  private ociKmsClient: KmsCryptoClient;

  constructor(sessionConfig: OCISessionConfig) {
    this.ociKmsClient = new KmsCryptoClient({ authenticationDetailsProvider: sessionConfig.getProvider() });
    this.ociKmsClient.endpoint = sessionConfig.getKmsEndpoint();
  }

  public getCryptoClient(): KmsCryptoClient {
    return this.ociKmsClient;
  }
}

File: sdk/python/storage/keeper_secrets_manager_storage/storage_oci_key_management/oci_kms_client.py
import logging
from .oci_session_config import OCISessionConfig

try:
    from oci.key_management import KmsCryptoClient
except ImportError:
    logging.getLogger().error("Missing OCI import dependencies."
                 " To install missing packages run: \r\n"
                 "pip install --upgrade \"oci\"\r\n")
    raise Exception("Missing import dependencies: oci")

class OciKmsClient:
    def __init__(self, session_config: OCISessionConfig):
        self.oci_kms_client = KmsCryptoClient(session_config.get_provider(), session_config.get_kms_endpoint())

    def get_crypto_client(self) -> KmsCryptoClient:
        return self.oci_kms_client
    